modules/Microsoft.VirtualMachineImages/imageTemplates/.test/common/deploy.test.bicep

```diff
@@ -58,35 +49,27 @@ module testDeployment '../../deploy.bicep' = {
     name: '<<namePrefix>>${serviceShort}001'
     customizationSteps: [
       {
-        restartTimeout: '30m'
+        restartTimeout: '15m'
         type: 'WindowsRestart'
       }
     ]
     imageSource: {
       offer: 'Windows-10'
       publisher: 'MicrosoftWindowsDesktop'
-      sku: '19h2-evd'
+      sku: 'win10-22h2-avd'
       type: 'PlatformImage'
       version: 'latest'
     }
-    userMsiName: resourceGroupResources.outputs.managedIdentityName
-    buildTimeoutInMinutes: 0
+    buildTimeoutInMinutes: 120
     imageReplicationRegions: []
-    lock: 'CanNotDelete'
-    managedImageName: '<<namePrefix>>-mi-${serviceShort}-001'
+    lock: ''
+    managedImageName: ''
     osDiskSizeGB: 127
-    roleAssignments: [
-      {
-        roleDefinitionIdOrName: 'Reader'
-        principalIds: [
-          resourceGroupResources.outputs.managedIdentityPrincipalId
-        ]
-        principalType: 'ServicePrincipal'
-      }
-    ]
+    roleAssignments: []
```

Comment:
Why no role assignment test?

Reply:
I moved this to the advanced test. When I left it in there I started getting 'clone' errors from the linter.

Reply:
Ideally, the common test should test as many parameters as possible. So I guess we 'could' merge the two if you're fine with that.
The Linter 'clone check' is honestly something to ignore. There are so many false positives in there that its not even worth bothering anymore - sadly. I guess it works better with classic binary code.

Reply:
+1 on merging common and advanced tests. Additional tests should be created only in case of conflicting parameters. The primary intention of tests is to validate parameters and not to provide resource use case examples, since, unfortunately, it would be difficult to commit to and to maintain that for all modules.

Reply:
I merged the tests